test-app/backend/package-lock.json

```diff
@@ -51,7 +51,6 @@
       }
     },
     "../shared": {
-      "name": "@general-vms/shared",
```

Comment:
Why?

Reply:
Not sure, it just does that when I rebuild pack-lock.json